tutorials/introductory/usage.py

```diff
@@ -205,7 +206,6 @@ def my_plotter(ax, data1, data2, param_dict):
 # which you would then use twice to populate two subplots:
 
 data1, data2, data3, data4 = np.random.randn(4, 100)  # make 4 random data sets
-xdata = np.arange(len(data1))  # make an ordinal for this
```

Comment:
we need `xdata` below.  It is better to define everything together.

Reply:
Sure, we need it here, were we also define `data`:
https://github.com/matplotlib/matplotlib/blob/505cdff57c9a2db4f4ac5a6cdd33c2d826a3fced/tutorials/introductory/usage.py#L384-L385
I think this looks much clearer to move the definition here, if we have to (re-)define data anyway:
```
xdata = np.arange(len(data1))
data = 10**data1
axs[0].plot(xdata, data)
```
(maybe re-defining `data` from a dict (line 117) to an array is not so good an idea, but this is another question)

Reply:
OK, I agree we can move it here.